ivy/array/set.py

```diff
@@ -50,4 +50,39 @@ def unique_all(
         return ivy.unique_all(self._data)
 
     def unique_inverse(self: ivy.Array) -> NamedTuple:
+        """
+          ivy.Array instance method variant of ivy.unique_inverse. This method simply
+          wraps the function, and so the docstring for ivy.unique_inverse also applies
+          to this method with minimal changes.
+
+          Parameters
+          ----------
+          self
+              input array. If ``x`` has more than one dimension, the function must flatten
+              ``x`` and return the unique elements of the flattened array.
+
+          Returns
+          -------
+          ret
+
+              a namedtuple ``(values, inverse_indices)`` whose
+
+              - first element must have the field name ``values`` and must be an array
+              containing the unique elements of ``x``. The array must have the same data
+              type as ``x``.
+              - second element must have the field name ``inverse_indices`` and must be an
+              array containing the indices of ``values`` that reconstruct ``x``. The array
+              must have the same shape as ``x`` and must have the default array index data
+              type.
+
+          Examples
+          --------
+
+        >>> x = ivy.array([0.3,0.4,0.7,0.4,0.2,0.8,0.5])
+        >>> y = ivy.unique_inverse(x)
```

Comment:
This way of calling the instance method is wrong! Please see your correct invocation of instance method in `ivy/container/set.py`!

Reply:
Ok thanks a lot. changed as requested by you.. just for info, i was getting same result when i called the method wrong way :P

Reply:
Yes, you would get the correct result because this way of calling the function is a functional call, whereas an instance method references the class of the variable itself.